README.md

```diff
@@ -45,86 +42,105 @@ Get the following:
    - [Linux](https://docs.docker.com/desktop/install/linux-install/)
 7. [NumPy](https://numpy.org/install/)
 
-> **TIP:** If you're a macOS user, you can also use `brew` to install NumPy:
+   > **TIP:** If you're a macOS user, you can also use `brew` to install NumPy:
 
-```bash
-brew install numpy
-```
+   ```bash
+   brew install numpy
+   ```
```

Comment:
```suggestion
   > ```bash
   > brew install numpy
   > ```
```

I missed that before, but it should be part of the TIP.

Reply:
Changed